rules/windows/defense_evasion_msbuild_making_network_connections.toml

```diff
@@ -1,8 +1,8 @@
 [metadata]
 creation_date = "2020/02/18"
-integration = ["endpoint", "windows"]
+integration = ["endpoint"]
 maturity = "production"
-updated_date = "2025/02/03"
+updated_date = "2025/02/21"
 min_stack_version = "8.14.0"
 min_stack_comments = "Breaking change at 8.14.0 for the Windows Integration."
```

Comment:
As we don't use the system integration anymore, we could remove the min_stack if that doesn't break our system.

Reply:
Should be fine for backports. Since it's min-stacked, related integrations field changes will only show up 8.14+ but since 8.13 is a fork, the rule differences shouldn't carry over to double bumps.

cc @shashank-elastic for verification.